go.mod

```diff
@@ -26,11 +26,15 @@ require (
 	go.mongodb.org/mongo-driver v1.11.6
 	go.uber.org/zap v1.24.0
 	golang.org/x/crypto v0.11.0 // indirect
-	golang.org/x/sys v0.10.0
+	golang.org/x/sys v0.12.0
 	google.golang.org/protobuf v1.30.0 // indirect
 )
 
-require github.com/yudai/golcs v0.0.0-20170316035057-ecda9a501e82 // indirect
+require (
+	github.com/aymanbagabas/go-osc52/v2 v2.0.1 // indirect
+	github.com/charmbracelet/lipgloss v0.9.1 // indirect
```

Comment:
installed but not used.

General suggestion would be to use more trusted libraries as many repos might have security vulnerability

Reply:
Thanks for advice, I forgot to remove them as I tried to show the logo as an image on the terminal using them.

Reply:
you can try to make the color more closer,with more accurate gradients.

Reply:
What about this?
![image](https://github.com/keploy/keploy/assets/76037906/2eebc05a-2c37-4569-96f2-a1df0a9b2376)

Reply:
something more resembling to keploy's logo would be possible too. the code snippet I shared was a sample of how to proceed. More refactoring can achieve the desired results.